doc/configuration.markdown

```diff
@@ -124,4 +124,13 @@ which will act as the cache key generator:
 	end
 	
 For more options see the [Rack::Request documentation](http://rack.rubyforge.org/doc/classes/Rack/Request.html)
-	
+
+### `use_native_ttl`
+
+Passes on the expiration timestamp to the cache store. This may be necessary
+with some stores to keep them from filling up, e.g. if using a Redis backend and
```

Comment:
> it should just be `store: Redis.new` or `Rails.cache` and done :D

I'd agree with you... mostly. We've been bitten by the fact the current Rack::Cache doesn't support dependency-injecting one's store (except with the [kludgy hack](https://github.com/rtomayko/rack-cache/blob/master/lib/rack/cache/storage.rb#L43-L44) for Memcache), because that causes 2 extra connections per process to the backend to be created (Redis, in our case). Not ideal scaling wise.

_But_, most users (at small scale) probably appreciate the simplicity of the current interface? As in, provide a URL and be done with it — no need to instantiate your own store.